src/libraries/NetCoreAppLibrary.props

```diff
@@ -145,6 +145,30 @@
       System.Xml.XPath;
       System.Xml.XPath.XDocument;
     </NetCoreAppLibrary>
+    <WindowsDesktopCoreAppLibrary>
+      Microsoft.Win32.Registry;
```

Comment:
This is wrong. Some of these libraries aren't part of WindowsDesktop's targeting pack anymore.